### PR TITLE
Fix camera blocking

### PR DESCRIPTION
**Pull request by bvanwijk**
The get methods in the IP camera class save the values for 2 seconds so this will reduce the amount of calls to the camera's.

### CODE REVIEW

**Comment by jochem725** [comment]
This seems like a good fix.

File: backend/src/test/java/com/benine/backend/camera/ipcameracontrol/IpcameraTest.java [hunk 1]
```diff
@@ -84,6 +84,25 @@ public final void testGetPosition() throws CameraConnectionException {
...
+    res = camera.getPosition();
+    assertEquals(0, res.getPan(), 0.000001);
+    assertEquals(180, res.getTilt(), 0.000001);
+  }
+  
+  @Test
+  public final void testGetPositionTwice3Seconds() throws CameraConnectionException, InterruptedException {
+    setCameraBehaviour("APC", "aPC80008000");
+    Position res = camera.getPosition();
+    Thread.sleep(2100);
```
Comment (Kingdorian) [inline backend/src/test/java/com/benine/backend/camera/ipcameracontrol/IpcameraTest.java]: This slows down testing by a lot if this happens to often.  Maybe you can make the amount of time before it times out an attribute (maybe even get it from the config) so you can set it to a really small amount when testing.

**Comment by Kingdorian** [comment]
Looks good apart from the little comment I made I think this can be merged.

**Comment by Kingdorian** [comment]
Great! I think we can merge this.

**Pull request merged**